### ISSUE TITLE
Feature: Display Recycle Bin as top level item in sidebar

### ISSUE THREAD

**Issue by Steperman5692** [files-community/Files#16037]
### What feature or improvement do you think would benefit Files?

So before I've used Files; I've used a program called Wintoys and in this program there was a option to add the Recycle Bin to Explorer navigation bar.
it look like this:
![image](https://github.com/user-attachments/assets/274b8250-c354-4c5a-87ba-6010e2d39c96)
So I was wondering if we could implement the recycle bin into Files in the navigation bar in Files

### Requirements

Add Recycle Bin to the navigation bar.
Add Recycle Bin in the right-click menu if the navigation bar.
I'm referring to this:
![image](https://github.com/user-attachments/assets/5ce1b0fd-b003-4016-8033-a83a73562b45)

### Files Version

3.6.0.0

### Windows Version

22631.4037

### Comments

Navigation Bar = Sidebar.
BTW I would be nice if we can move the sidebar items (WSL, Drives...) whatever we want in the sidebar.

**Comment by Steperman5692** [comment]
Yeah ik but I would be nice if we put the recycle bin in a seperate section instead of pinning it.

**Comment by yaira2** [comment]
Can you share more about your usecase for having Recycle Bin as a top level item?

**Comment by Steperman5692** [comment]
It's easier to access the recycle bin especial when you click on the recycle bin on the desktop when the "Set Files as the default file manager" flag is turn off (I will open file explorer if you did that) + if you open file explorer and the recycle bin is pinned and it's also added the navigation bar (using Wintoys) it will look weird...
I'm referring to this:
![image](https://github.com/user-attachments/assets/b2616687-9755-4123-b87a-9f65903dc6fb)
